### DISCUSSION TITLE
The additions to the .vscode settings file

### DISCUSSION THREAD

**Discussion by Fasani** [estruyf/vscode-front-matter#93] [Q&A] [answered]
I know some people suggest against committing the `.vscode` folder/files. I like to commit it, when it is only myself working on a project. I wondered if the file paths below could be relative instead of absolute then when I switch machines I have no issues, but of course, this is just an opinion.

```
{
    "frontMatter.content.folders": [
        {
            "fsPath": "/Users/fasani/Projects/michael-fasani-gatsby/src/posts",
            "paths": [
                "/Users/fasani/Projects/michael-fasani-gatsby/src/posts",
                "\\Users\\fasani\\Projects\\michael-fasani-gatsby\\src\\posts"
            ]
        }
    ]
}
```

**Comment by estruyf** [comment, marked as answer]
@Fasani in the upcoming version I changed this indeed to a relative path. Originally this was implemented to support WSL. As paths there are a bit tricky, you got the Windows one and Linux one.

In the upcoming version, this setting will automatically be converted to the new one: [frontmatter.content.pagefolders](https://beta.frontmatter.codes/docs/settings#frontmatter.content.pagefolders)
